lib/OpenQA/Schema/Result/Comments.pm

```diff
@@ -42,8 +42,9 @@ __PACKAGE__->add_columns(
         data_type      => 'integer',
         is_foreign_key => 1,
     },
-    hidden => {
-        data_type     => 'boolean',
+    flags => {
```

Comment:
Might be - but I don't think we will be able to find the discussion in #837 when looking for a why